sections/main-product.liquid

```diff
@@ -281,6 +293,36 @@
                     </span>
                   {%- endif -%}
                 </div>
+                {%- if product.quantity_price_breaks_configured? -%}
+                  <volume-pricing class="parent-display" id="Volume-{{ section.id }}">
+                    <span class="caption-large">{{ 'products.product.volume_pricing.title' | t -}}</span>
+                    <ul class="list-unstyled">
+                      <li>
+                        <span>{{ product.selected_or_first_available_variant.quantity_rule.min }}+</span>
+                        <span>{{ product.selected_or_first_available_variant.price | money_with_currency }}/ea</span>
+                      </li>
+                      {%- for price_break in product.selected_or_first_available_variant.quantity_price_breaks -%}
+                        <li class="{%- if forloop.index >= 3 -%}show-more-item hidden{%- endif -%}">
+                          <span>{{ price_break.minimum_quantity }}<span aria-hidden="true">+</span></span>
```

Comment:
We dont do this here: https://github.com/Shopify/dawn/blob/main/snippets/facets.liquid#L295. Could we do this in a follow up PR and update both so this PR doesnt rely on new translations?

Reply:
@KaichenWang Thinking about this as Im creating the issue and I think this makes sense as is so we can ensure this span is aria-hidden true. 

Instead of changing it to:
```
{{ 'products.facets.show_more' | t -}}</span>

"each": "Show more + "
```
thoughts?

Reply:
In this case, the `+` shouldn't be hidden for screenreaders. The `+` should be part of the string. For example, "6" and "6+" have different meaning to the buyer.